csharp/ql/src/semmle/code/csharp/frameworks/ServiceStack.qll

```diff
@@ -78,7 +78,21 @@ module Sources {
         }
     }
 }
-
+/** Flow Sinks for the ServiceStack framework */
+module Sinks {
+    private import semmle.code.csharp.security.dataflow.flowsinks.Remote
+  
+    /** RemoteFlow sinks for service stack */
+    class ServiceStackRemoteRequestParameter extends RemoteFlowSink {
+      ServiceStackRemoteRequestParameter() {
+        exists(MethodCall mc |
+          mc.getTarget().hasQualifiedName("ServiceStack.IRestClient.Get") and
+          mc.getArgument(0) = this.asExpr()
+        )
+      }
+    }
+  }
```

Comment:
Do you expect new findings due to this change? `RemoteFlowSink` is not used in that many checks. Also, I find that `ServiceStack.IRestClient.Get` falls into the same category as a simple `HttpClient` `GET` call, yet, I think the latter is not covered in `RemoteFlowSinks`. By the way, https://github.com/github/codeql/pull/5110 is somewhat related, it reports on server side requests that might access untrusted sites.

Reply:
What kind of sink should it inherit?  ExternalLocation?  Or just plain-old sink?